packages/components/src/Tabs/src/TabList.tsx

```diff
@@ -0,0 +1,84 @@
+import { useStyledSystem, type StyledComponentProps } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, useContext, useLayoutEffect, useState, type CSSProperties, type ForwardedRef } from "react";
+import { TabList as RACTablist, TabListStateContext, useContextProps, type TabListProps as RACTablistProps } from "react-aria-components";
+
+import { cssModule, type BaseComponentDOMProps } from "../../utils/index.ts";
+
+import { TabLine } from "./TabLine.tsx";
+import { InternalTabsContext, TabListContext } from "./TabsContext.ts";
+
+import styles from "./TabList.module.css";
+
+export const GlobalTabListCssSelector = "hop-TabList";
+
+export interface TabListProps<T> extends
+    Omit<RACTablistProps<T>, "children" | "className" | "style">,
+    StyledComponentProps<BaseComponentDOMProps> {}
+
+function TabList<T extends object>(props: TabListProps<T>, ref: ForwardedRef<HTMLDivElement>) {
+    [props, ref] = useContextProps(props, ref, TabListContext);
+    const { variant, ...lineProps } = useContext(InternalTabsContext) ?? {};
```

Comment:
I think it's safer here to deconstruct the props the line needs and forward it, rather then forwarding everything. imagine if we had aria-label, it wouldn't make sense to forward id